Sources/Public/MonthsLayout.swift

```diff
@@ -116,3 +128,41 @@ public struct VerticalMonthsLayoutOptions: Equatable {
   public let alwaysShowCompleteBoundaryMonths: Bool
 
 }
+
+// MARK: - HorizontalMonthsLayoutOptions
+
+/// Layout options for a horizontally-scrolling calendar.
+public struct HorizontalMonthsLayoutOptions: Equatable {
+
+  // MARK: Lifecycle
+
+  /// Initializes a new instance of `HorizontalMonthsLayoutOptions`.
+  ///
+  /// - Parameters:
+  ///   - numberOfFullyVisibleMonths: The maximum number off fully visible months for any scroll offset. The default value is
+  ///   `1`.
+  public init(numberOfFullyVisibleMonths: Double = 1) {
+    assert(numberOfFullyVisibleMonths >= 1, "`numberOfFullyVisibleMonths` must be greater than 1.")
+    self.numberOfFullyVisibleMonths = numberOfFullyVisibleMonths
+  }
+
+  // MARK: Public
+
+  /// The maximum number off fully visible months for any scroll offset.
+  public let numberOfFullyVisibleMonths: Double
```

Comment:
So, is this a maximum or will you always get the provided value? It's unclear to me from the naming of this variable and documentation that makes me think something differently.

Reply:
@brynbodayle it's the "maximum" in the sense that there can never be more than this number of _fully_ visible months for any scroll position. 

For example, if you set this to 2, then no matter what scroll position you're at, you'll at most see 2 _fully visible_, un-clipped months. However, you might see more than 2 _partially_ visible months for certain scroll positions, like this:

<img width="510" alt="image" src="https://user-images.githubusercontent.com/746571/98587718-9e267700-226e-11eb-8afd-c8717d1a9cf1.png">

Do you have any naming suggestions? I'll think on it some more too.

Reply:
ah gotcha, how about we call it `maximumFullyVisibleMonths` then?

Reply:
Yeah that sounds good to me too, will update!